custom_components/tuya_local/devices/th05_temp_humidity_sensor.yaml

```diff
@@ -0,0 +1,44 @@
+name: TH05-BLE Temperature and Humidity Sensor
+products:
+  - id: vyfoip9h
+primary_entity:
+  entity: sensor
+  class: temperature
+  dps:
+    - id: 1
+      name: sensor
+      type: integer
+      unit: C
+      mapping:
+        - scale: 10
+      class: measurement
+secondary_entities:
+  - entity: sensor
+    class: humidity
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: select
+    name: Display Units
+    category: config
+    dps:
+      - id: 9
+        name: option
+        type: string
+        mapping:
+          - dps_val: "f"
+            value: Fahrenheit
+          - dps_val: "c"
+            value: Celsius
+        optional: true
```

Comment:
If this affects the value of dp 1, then dp 9 needs to also be listed under the sensor entity with name "unit" instead of a fixed unit on dp 1.  In that case, "Temperature unit" would be a better name, as it does not just affect display.

Reply:
Because of the original choice of name, I'll assume this does not apply.